scripts/src/config.ts

```diff
@@ -4,6 +4,7 @@ import "source-map-support/register";
 import { path } from "./utils";
 
 import { LogTarget } from "./logging";
+import { SignInType } from "./models/applications";
```

Comment:
**NO NO NO !!!**

the config must know of no other module that might depend on him later on, we want to avoid circular dependencies.
so, either define `SignInType` here and use it in the  `models/applications` module, or create a new module which holds the types and then both will use that.

Reply:
I think I'll just define it as plain string. There is no enforcement here anyway.